### PR TITLE
Jmm/include findings

### PR DESCRIPTION
**Pull request by JeanMarie-TTS**
- Add support for Findings, FindingText, CapText
- loader report summary at the end. Stops after 25 errors

### CODE REVIEW

File: backend/c2g/workbooklib/trabsformers.py
Comment (eazapata23) [inline backend/c2g/workbooklib/trabsformers.py:1]: 1. The file named is misspelled 'trabsformers.py' should be 'transformers.py' 
2. L 4 'get_cpapuntry' is also misspelled should be 'get_cpacountry'

File: backend/c2g/README.md [hunk 1]
```diff
@@ -36,10 +37,33 @@ manage.py raw_to_pg --clean True
 
 * data A folder that contains sample data that we can use for development.
 
+* wb_generator.py This module loads a single submission from the history tables to the GSA FAC tables
+
+* loader.py This module will eventually loadd all of the historic data by invoking wb_generator for each submission
```
Comment (eazapata23) [inline backend/c2g/README.md:42]: L 42 'loadd' should be 'load'

**Review by eazapata23 — COMMENTED** [review]
Found a few misspellings

**Review by phildominguez-gsa — APPROVED** [review]

**Review by eazapata23 — APPROVED** [review]
Conflicts resolved.

**Pull request merged**